src/modules/map/i18n/map.provider.js

```diff
@@ -20,7 +21,8 @@ export const provide = (lang) => {
 				map_info_button: 'Information',
 				map_info_button_help: 'Hilfe',
 				map_info_button_contact: 'Kontakt',
-				map_info_button_about: 'Impressum'
+				map_info_button_about: 'Impressum',
+				map_zoom_extent_button: 'Ganz Bayern anzeigen'
```

Comment:
we can avoid a country-specific text?

Reply:
I think, it's OK when we use BayernAtlas specific translations in German